### PR TITLE
Add script to check default values have been persisted on all objects.

### PR DESCRIPTION
**Pull request by njohner**
This is based on the `find_non_persisted_values.py` script.

### CODE REVIEW

File: opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        self.stats['by_field'] = Counter()
+
+        ts = datetime.today().strftime('%Y-%m-%d_%H-%M-%S')
+        self.csv_log_path = self.get_logfile_path(
+            'find-nonpersistent-values-%s.csv' % ts)
+        self.summary_log_path = self.get_logfile_path(
+            'find-nonpersistent-values-summary-%s.log' % ts)
+
+    def run(self):
+        sys.stderr.write("Checking for non-persisted values...\n\n")
```
Comment (deiferni) [inline opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py:51]: `stderr` does not feel correct here

File: opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                for i, brain in enumerate(all_brains):
+                    obj = brain.getObject()
+                    missing_fields = self.check_for_missing_fields(obj)
+                    self.update_stats(missing_fields)
+
+                    if missing_fields:
+                        self.write_csv_row(obj, missing_fields)
+
+                    if i % 100 == 0:
+                        sys.stderr.write("Progress: %s of %s objects\n" % (i, total))
```
Comment (deiferni) [inline opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py:69]: i would prefer a `logger` here, there are examples in other scripts how to set one up that is working. directly writing to stdout/stderr is a bit of an antipattern IMO.

**Review by deiferni — APPROVED** [review]
Nothing wrong with the logic IMO, only some nagging about the logging.

File: opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py (hunk 2 above)
Comment (njohner) [reply opengever/maintenance/scripts/find_fields_with_non_persisted_default_values.py:69]: Right. I copied the existing script and made minimal adjustments to the logic to only get non persisted fields that actually have a default value. But you're right I'll change that

**Pull request merged**